src/views/portfolio/projects/FindingAudit.vue

```diff
@@ -0,0 +1,246 @@
+<template>
```

Comment:
Aside from the imports and structure, this is all just a copy-and-paste from the inline version.